ServerCore/Pages/Events/Edit.cshtml

```diff
@@ -120,6 +120,14 @@
                 <input asp-for="EditableEvent.DefaultLunch" class="form-control" />
                 <span asp-validation-for="EditableEvent.DefaultLunch" class="text-danger"></span>
             </div>
+            <div class="form-group">
+                <label asp-for="EditableEvent.LunchOptions" class="control-label"></label>
+                <textarea asp-for="EditableEvent.LunchOptions" class="form-control"></textarea>
+                <span asp-validation-for="EditableEvent.LunchOptions" class="text-danger"></span>
+                <div style="font-size:11px;">Each option is composed of a "Lunch Name":"Lunch Description"; pair, including the quotes. Each option is delimited by a ; semicolon, so whitespace outside of quotes is ignored. COLON AND SEMICOLON CANNOT BE USED INSIDE THE QUOTES.</div>
+                <div style="font-size:11px;">The Lunch Name is also used as the value that's written to the database. Lunch Name == "noneoftheabove" after removing spaces and converting to lowercase will make the "Custom Order" textbox appear on the Swag Registration page for events with single-player lunches.</div>
+            </div>
```

Comment:
Can you add a display of the current parse of LunchOptions here, so we can catch errors before players do?